server/auth/nop.go

```diff
@@ -18,18 +18,18 @@ import (
 	"context"
 )
 
-type tokenNop struct{}
+type TokenNop struct{}
```

Comment:
Don't expose internal testing code from package! Use proper TokenProvider in external tests.

Reply:
reverted